testing/sensortest/sensortest.c

```diff
@@ -178,6 +180,16 @@ static void print_ppgq(const char *buffer, const char *name)
          event->gain[2], event->gain[3]);
 }
 
+static void print_cap(FAR const char *buffer, FAR const char *name)
+{
+  struct sensor_cap *event = (struct sensor_cap *)buffer;
```

Comment:
```suggestion
  FAR struct sensor_cap *event = (FAR struct sensor_cap *)buffer;
```
But I see that many places across the file are not equipped with FAR.

Reply:
Done.